src/components/NotAuthorized.tsx

```diff
@@ -0,0 +1,92 @@
+import { Split, SplitItem, StackItem } from '@patternfly/react-core';
+import { Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { NotAuthorized } from '@redhat-cloud-services/frontend-components';
+import { getInsights, localUrl } from '@redhat-cloud-services/insights-common-typescript';
+import { default as React, useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import { useLocation, useParams } from 'react-router-dom';
+
+import Config from '../config/Config';
+import messages from '../properties/DefinedMessages';
+import { useGetBundles } from '../services/Notifications/GetBundles';
+import { Facet } from '../types/Notification';
+import { getSubApp } from '../utils/Basename';
+
+interface NotificationListPageParams {
+    bundleName: string;
+}
+
+export const NotAuthorizedPage: React.FunctionComponent = () => {
+
+    const { bundleName } = useParams<NotificationListPageParams>();
+    const getBundles = useGetBundles();
+    const bundles: Facet | undefined = useMemo (() => {
+        if (getBundles.payload?.status === 200) {
+            return getBundles.payload.value.find(b => b.name === bundleName);
+        }
+    }, [ bundleName, getBundles.payload?.status, getBundles.payload?.value ]);
```

Comment:
This is not yet going to work - but is something we can update.
IIRC facets require read permissions - we could update it so that it does not require any permission.

Reply:
ok, yeah, after those fixes this no longer is showing. Would I be looking at the generated files for this change?

Reply:
nevermind my comment above - this api doesn't require any permission

Reply:
hmmm.. its coming up undefined now, where it wasn't before.

Reply:
This is because we don't have a router in there -  we can't use the `useParams` as there is nothing to match against.

Reply:
yep, spoke too soon, just looked through and saw that.